src/tests/ftest/util/telemetry_test_base.py

```diff
@@ -264,6 +271,33 @@ def sum_values(metric_out):
 
         return total
 
+    def secure_server_telemetry_setup(self):
+        """ Setup secure server certificate for telemetry."""
+        self.log.info("Secure Server Telemetry Setup start")
+
+        # Create the Certificate
+        self.server_managers[0].prepare_telemetry_certificate()
+        yaml_data = self.server_managers[0].manager.job.yaml.get_yaml_data()
+
+        # Update the certificate in yaml dictionary.
+        https_cert = os.path.join(self.server_managers[0].telemetry_certificate_dir,
+                                  "telemetry.crt")
+        https_key = os.path.join(self.server_managers[0].telemetry_certificate_dir,
+                                 "telemetry.key")
+        yaml_data["telemetry_config"].update({"https_cert": https_cert})
+        yaml_data["telemetry_config"].update({"https_key": https_key})
+
+        # Update the current yaml file.
+        self.server_managers[0].manager.job.create_yaml_file(yaml_data)
+
+        # Restart the DAOS servers
+        self.log.info("Stop DAOS servers")
+        self.server_managers[0].manager.stop()
+        self.log.info("Start daos_server and detect the DAOS I/O engine message")
+        self.server_managers[0].restart(hosts=self.hostlist_servers)
```

Comment:
This seems really heavy just to setup telemetry? Ideally this should be done before starting

Reply:
If you put this in the test config, each mux variant will start and stop servers and agents
```
setup:
  start_servers_once: False
  start_agents_once: False
```

Reply:
@phender ?